src/core/style/layout.scss

```diff
@@ -153,37 +150,43 @@
   // (0-20) * 5 = 0-100%
   @for $i from 0 through 20 {
     [#{$flexName}="#{$i * 5}"] {
-      flex: 0 0 #{$i * 5 + '%'};
+      flex: #{($i * 5)/100};
     }
     [layout="row"] > [#{$flexName}="#{$i * 5}"] {
       max-width: #{$i * 5 + '%'};
+      min-width: #{$i * 5 + '%'};
     }
     [layout="column"] > [#{$flexName}="#{$i * 5}"] {
       max-height: #{$i * 5 + '%'};
+      min-height: #{$i * 5 + '%'};
     }
   }
 
   [#{$flexName}="33"], [#{$flexName}="34"] {
-    flex: 0 0 33.33%;
+    flex: 0.33;
```

Comment:
Nested layouts do **not** flex properly unless the shorthand, decimal version is used:

See [Nested Flex Containers](http://codepen.io/team/AngularMaterial/pen/OVOZMw/) demo.